packages/headless/src/utils/query-expression/date-field/date-field.ts

```diff
@@ -3,8 +3,19 @@ import {getOperatorSymbol, NumericOperator} from '../common/operator';
 import {Part} from '../common/part';
 
 export interface DateFieldExpression extends Negatable {
+  /**
+   * The field name.
+   */
   field: string;
+
+  /**
+   * The operator to use to compare the `field` and `value`.
+   */
   operator: NumericOperator;
```

Comment:
If there's doc for it, I think we should link doc in every `operator` parameter.